src/Texts/Texts.js

```diff
@@ -0,0 +1,10 @@
+const texts = {
+  logs: {
```

Comment:
please change `logs.errors` to errorMesseges

Reply:
moved only this error text to AppUtils, in order to be able to individually export texts's text categories, due to IE Breaking if texts had exports which wasn't default.